src/components/sbb-pearl-chain-vertical/sbb-pearl-chain-vertical.stories.js

```diff
@@ -0,0 +1,687 @@
+import { h } from 'jsx-dom';
+import readme from './readme.md';
+
+const lineType = {
+  options: ['dotted', 'standard', 'thin'],
+  control: { type: 'radio' },
+};
+const lineColor = {
+  options: ['charcoal', 'red', 'metal', 'sky'],
+  control: { type: 'radio' },
+};
+const dotType = {
+  options: ['standard', 'thin-bullet', 'thick-bullet', 'double-bullet'],
+  control: { type: 'radio' },
+};
+const dotColor = {
+  options: ['charcoal', 'red', 'metal', 'sky'],
+  control: { type: 'radio' },
+};
+const dotSize = {
+  options: ['small', 'medium', 'large'],
+  control: { type: 'radio' },
+};
+
+const hideLine = {
+  control: {
+    type: 'boolean',
+  },
+};
+
+const minHeight = {
+  control: { type: 'number' },
+};
+
+const position = {
+  control: { type: 'number' },
+};
+const defaultArgTypes = {
+  lineType,
+  lineColor,
+  dotType,
+  dotColor,
+  dotSize,
+  hideLine,
+  minHeight,
+  position,
+};
+
+const defaultArgs = {
+  lineType: lineType.options[1],
+  lineColor: lineColor.options[0],
+  dotType: dotType.options[0],
+  dotColor: dotColor.options[0],
+  minHeight: '100',
+  hideLine: false,
+  dotSize: dotSize.options[1],
+  position: 0,
+};
+
+const Template = (args) => {
+  return (
+    <sbb-pearl-chain-vertical>
+      <sbb-pearl-chain-item pearlChainItemAttributes={args}>
+        <div slot="right" style={{ marginTop: '-10px', marginInlineStart: '10px' }}>
```

Comment:
Looks a little bit strange to me, that margins are applied to  the div here. Is this on purpose? If yes, maybe a short comment?

Or should this not be part of the component itself?

Reply:
It is not part of the component, because the pearlchain itself will be used in various ways with different spancing/styling.

Reply:
Ok, but then I recommend you to use css vars (and maybe use padding then inside the component).